### ISSUE TITLE
Relationship FU Plan: Steve Leist

### ISSUE THREAD

**Issue by jaahmuhl** [promazo/100k-Mentor-Challenge-Outreach#153]
**User Story / Problem / Details**
The Content Editor is in charge of ensuring the initial relationship we foster with an interviewee does not decay over time for interviewees that we designate as important. As a result we want to follow a process that allows us to interact and build a relationship with the interviewee over time. The final step of the process should be focused on how connect with an interviewee over multiple interactions and over the course of a year.

**Technical Definition of Done**
Complete all steps in the process

**Definition of Ready**
All of the steps in the process have been approved by Brett 

**Process**
- [ ] Review the interview in its entirety 
- [ ] Thank you note to the participant
- [ ] Quick clip (30s) that shows the interviewee some of the raw footage/a particularly interesting part of the discussion
- [ ] ID potential contacts
- [ ] Formulate ask
- [ ] Create a Relationship Action Plan

**Comment by howie-emond** [comment]
Hi, Steve, we wanted to reach out and thank you again for participating in our Executive Interview Series. During our discussion, we had the privilege of hearing about your career journey becoming a leader in IT, your views on the future of the workplace, advice on how to grow as a leader, and tips for students entering the industry. Both the Promazo team and our student audience will be excited to learn from you and your experiences.

Attached is a short clip from the interview that we found particularly interesting. The points you made on making sure you find enjoyment both within your career and with your team members was an important point for our students to understand.

We are curious to hear if there are any opportunities to conduct similar interviews with some of your colleagues. We have identified the following as potential targets for future interviews: 
Maya Leibman
Derek Kerr
Vasu Raja
David Seymour
Cole Brown

Any contacts for these individuals, or other recommendations you may have, would benefit our project immensely.

Also, if you have any time in the future, we would be thrilled to have you on for a second interview, as your input and advice has been greatly appreciated. A discussion which dives further into your personal journey and visions on the industry’s future would be meaningful to our students.

Again, we thank you for your time and meaningful conversation. We look forward to building a closer relationship with you.

**Comment by howie-emond** [comment]
ATTATCHED CLIP
- In American Airlines folder 
(2 folders for Leist apparently, "American Airlines" and the regular "American Airlines-Steve Leist-...")
- American Airlines Example 1 IG .mp4

https://drive.google.com/drive/u/1/folders/1PLh0bD1UL6SRV2d3c2u57Yzd7GPS8nnr

**Comment by howie-emond** [comment]
Profile Links

Maya Leibman
https://www.linkedin.com/in/maya-leibman-88a863188/

Derek Kerr
https://www.linkedin.com/in/derek-kerr-5105585/

Vasu Raja
https://www.linkedin.com/in/vasu-raja-a48417/

David Seymour
https://www.linkedin.com/in/davidgseymour/

Cole Brown
https://www.linkedin.com/in/cole-brown-0b43b395/

**Comment by howie-emond** [comment]
UPDATED MESSAGE

Hi Steve, we wanted to reach out and thank you again for participating in our Executive Interview Series.

During our discussion, we had the privilege of hearing about your career journey becoming a leader in IT, your views on the future of the workplace, advice on how to grow as a leader, and tips for students entering the industry. Both the Promazo team and our student audience will be excited to learn from you and your experiences.

Attached is a short clip from the interview that we found particularly interesting. The points you made on making sure you find enjoyment both within your career and with your team members was an important point for our students to understand.

As we closed out the interview, we discussed having a second conversation to dive deeper into your concepts and strategies. When your schedule permits, we would greatly appreciate having you on for another 30 minutes. Again, we thank you for your time and conversation, and look forward to building a closer relationship with you.

SAME CLIP
